pkgs/by-name/du/dust/package.nix

```diff
@@ -1,14 +1,16 @@
 { stdenv, lib, fetchFromGitHub, rustPlatform, AppKit, installShellFiles }:
 
 rustPlatform.buildRustPackage rec {
+  # The attribute name has been renamed to `dust`.
+  # See https://github.com/NixOS/nixpkgs/pull/282233.
   pname = "du-dust";
```

Comment:
```suggestion
  # Originally, this package was under the attribute `du-dust`, since `dust` was taken.
  # Since then, `dust` has been freed up, allowing this package to take that attribute.
  # However in order for tools like `nix-env` to detect package updates, keep `du-dust` for pname.
  pname = "du-dust";
```